src/models/CartProduct.ts

```diff
@@ -0,0 +1,19 @@
+class CartProduct {
```

Comment:
rename to Product
use interface and export it right away (do it everywhere ... do not use export default):
export interface Product { ...}